recipes/r-downloader/meta.yaml

```diff
@@ -0,0 +1,44 @@
+package:
+  name: r-downloader
+  version: 0.0.4
+
```

Comment:
Thanks @bgruening. I have updated all my PRs with the meta.yaml files generated using the `conda_r_skeleton_helper` as you suggested me here [#3399-issuecomment-320343694](https://github.com/conda-forge/staged-recipes/pull/3399#issuecomment-320343694)